docs/faq.md

```diff
@@ -0,0 +1,89 @@
+# Frequently Asked Questions
+
+## What is Jupytext?
+
+Jupytext is a Python package that can convert Jupyter notebooks to scripts or Markdown documents. It can also convert these text documents back to Jupyter notebooks.
+
+## Why would I want to convert my notebooks to text?
+
+The text representation have much cleaner diffs than the original notebook format. Merging multiple contributions to a notebook in any of these text formats is easier than with the JSON format. Last but not least, acting on a notebook represented as text (spell check, reformat, ...) is sometimes more comfortable than in Jupyter.
+
+## How do I use Jupytext?
+
+Open the notebook you want to version control. _Pair_ the notebook to a script or a Markdown file using either the [Jupytext Menu](https://github.com/mwouts/jupytext/blob/master/README.md#jupytext-menu-in-jupyter-notebook) in Jupyter Notebook or the [Jupytext Commands](https://github.com/mwouts/jupytext/blob/master/README.md#jupytext-commands-in-jupyterlab) in JupyterLab.
+
+Save the notebook, and you get two copies of the notebook: the original `*.ipynb` file, together with its paired text representation.
+
+## Which Jupytext format do you recommend?
+
+I tend to use the Markdown format for notebooks that contain more text than code, as Markdown documents are conveniently edited in IDEs and also well rendered on GitHub.
```

Comment:
I'd avoid using the word "I" in package documentation, I usually use "we" in my packages even if I'm largely the one developing the package (the hope is always that one day the package developer community will be a "we" rather than an I one day :-) )